cylc/flow/parsec/fileparse.py

```diff
@@ -239,27 +268,63 @@ def read_and_proc(fpath, template_vars=None, viewcfg=None, asedit=False):
         flines = inline(
             flines, fdir, fpath, False, viewcfg=viewcfg, for_edit=asedit)
 
+    template_vars['CYLC_VERSION'] = __version__
+
     # process with EmPy
     if do_empy:
+        if (
+            extra_vars['templating detected'] == 'empy:suite.rc' and
+            not re.match(r'^#![Ee]m[Pp]y\s*', flines[0])
+        ):
+            if not re.match(r'^#!', flines[0]):
+                flines.insert(0, '#!empy')
+            else:
+                raise FileParseError(
+                    "Your file has shebang line incompatible with empy"
+                    " variables."
```

Comment:
Highly unlikely anyone will hit this but could make it a little more informative, say:

```suggestion
                    f"Your file has shebang line for {extra_vars['templating_detected']}"
                    f" but {flines[0]} configured in flow.cylc.
```

Same for Jinja2 equiv.

(untested)

Reply:
Dun.